src/utils/channel_utils.js

```diff
@@ -218,7 +220,16 @@ export function isGroupOrDirectChannelVisible(channel, memberships, config, myPr
     return isDirectChannelVisible(users[otherUserId] || otherUserId, config, myPreferences, channel, lastPost, isUnreadChannel(memberships, channel));
 }
 
-export function showCreateOption(config, license, channelType, isAdmin, isSystemAdmin) {
+export function showCreateOption(state, config, license, teamId, channelType, isAdmin, isSystemAdmin) {
```

Comment:
Do we need to pass in most of these when we just have the state?

Reply:
0/5 on that. Changing this will require rewriting all the unit tests that have been added, so it's not likely not worth it at this point

Reply:
Yes, probably we can rewrite it later.